### PR TITLE
Go to the next line when cursor is in a wrapped line

### LINKED ISSUE #8: Cannot move cursor to the next line when cursor is in a wrapped line
When a long line wraps around in VS editor cursor won't move past it. It'll get stuck there.

### PR DESCRIPTION
**Pull request by mikekov**
This is a simple fix for cursor getting stuck in a wrapped line. It'll move to the next logical line when key down is pressed.

fixes #8

### CODE REVIEW

File: TabSanity/ArrowKeyFilter.cs [hunk 1]
```diff
@@ -104,8 +104,9 @@ public override int Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt,
 					case ARROW_DOWN:
 						try
 						{
-							_snapshotLine = TextView.TextSnapshot.GetLineFromPosition(
-								Caret.Position.BufferPosition.Add(CaretLine.Length - CaretColumn + 2));
+//							_snapshotLine = TextView.TextSnapshot.GetLineFromPosition(
```
Comment (jedmao) [inline TabSanity/ArrowKeyFilter.cs]: I won't be merging in commented out code. This is what version control is for. Go ahead and just delete these 2 lines.

File: TabSanity/ArrowKeyFilter.cs [hunk 2]
```diff
@@ -119,6 +120,15 @@ public override int Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt,
 			return NextTarget.Exec(ref pguidCmdGroup, nCmdID, nCmdexecopt, pvaIn, pvaOut);
 		}
 
+		ITextSnapshotLine FindNextLine()
+		{
+			var current = TextView.TextSnapshot.GetLineFromPosition(Caret.Position.BufferPosition.Position);
```
Comment (jedmao) [inline TabSanity/ArrowKeyFilter.cs]: Please cache `TextView.TextSnapshot` and `Caret.Position.BufferPosition` at the top of this function for reuse.

```cs
var snapshot = TextView.TextSnapshot;
var caretBufferPosition = Caret.Position.BufferPosition;
```

**Comment by jedmao** [comment]
Thanks for doing the dirty work! Address these issues and I'll merge it in.

**Comment by mikekov** [comment]
Thanks for review. I just fixed it, tested and pushed new commit.

**Pull request merged**